lib/puppet/provider/openldap_overlay/olc.rb

```diff
@@ -111,12 +111,15 @@ def create
       end
     end
     t.close
-    Puppet.debug(IO.read t.path)
+    ldif_content = IO.read t.path
+    Puppet.debug(ldif_content)
     begin
       ldapmodify('-Y', 'EXTERNAL', '-H', 'ldapi:///', '-f', t.path)
     rescue Exception => e
       raise Puppet::Error, "LDIF content:\n#{IO.read t.path}\nError message: #{e.message}"
     end
+
+    ldif_content
```

Comment:
To be able to test the function. I cannot introspect anything it does otherwise. See for example: https://github.com/camptocamp/puppet-openldap/pull/128/files#diff-208882fb16538ba00ae5fbedc6bece85R46